qiskit/chemistry/qubit_transformations/qubit_operator_transformation.py

```diff
@@ -24,8 +24,9 @@ class QubitOperatorTransformation(ABC):
     """Base class for transformation to qubit operators for chemistry problems"""
 
     @abstractmethod
-    def transform(self, driver: BaseDriver
-                  ) -> Tuple[WeightedPauliOperator, List[WeightedPauliOperator]]:
+    def transform(self, driver: BaseDriver,
+                  additional_operators: Optional[Dict[str, Any]] = None
+                  ) -> Tuple[WeightedPauliOperator, Dict[str, WeightedPauliOperator]]:
         """transforms to qubit operators """
```

Comment:
We should probably extend this docstring. Most of the `FermionicTransformation` should be good to use though.

Reply:
I tried to update it, can you check it you agree?

Reply:
Lgtm :+1: